client/js/app/validations/ExplorerValidations.js

```diff
@@ -46,6 +46,20 @@ module.exports = {
 
   },
 
+  percentile_value: {
+    
+    msg: 'Choose a Percentile Value.',
+
+    shouldRun: function(model) {
+      return model.query.analysis_type === 'percentile';
+    },
+    
+    validate: function(model) {
+      return typeof model.query.percentile === 'number';
```

Comment:
Checking for existence sounds like a good thing to do here, thanks for the suggestion. Since we currently use `null` (set in the store) and `''` (in the common input component) as default values, I'd advocate for an explicit return statement that also checks whether the property value is a `Number` type. What do you think of this approach?

`return model.query.percentile !== null && model.query.percentile !== '' && typeof model.query.percentile === 'number';`